_includes/specials/taproot/00-bech32m.md

```diff
@@ -0,0 +1,42 @@
+## Preparing for taproot #1: bech32 sending support
+{% assign trb = "709,632" %}{% comment %}<!-- stop me from mistyping the number -->{% endcomment %}
```

Comment:
Perhaps make this a site-wide alias? I'm sure we'll be referring to 709,632 in many places over the next few months.